OpenFOAM/bike_OpenFOAM_v11.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+module load OpenFOAM/11-foss-2023a
```

Comment:
We have a `module load` here, but not in `OpenFOAM_v8.sh`

I think I prefer to have it in here rather than in `run.sh` though...